include/swift/AST/DiagnosticsClangImporter.def

```diff
@@ -388,5 +384,15 @@ NOTE(nonescapable_field_of_escapable, none,
      "escapable record %0 cannot have non-escapable field '%1'",
      (const clang::NamedDecl *, StringRef))
 
+GROUPED_WARNING(
+    invalid_conditional_attr, ClangDeclarationImport, none,
+    "swift attribute '%0' ignored because it is only supported in templates",
```

Comment:
Can we say "class templates" instead of "templates" to be a tiny bit more specific?